hera_sim/interpolators.py

```diff
@@ -4,7 +4,26 @@
 
 import numpy as np
 from cached_property import cached_property
-from scipy.interpolate import RectBivariateSpline
+from scipy.interpolate import RectBivariateSpline, interp1d
+from hera_sim.data import DATA_PATH
+from os import path
+
+INTERP_OBJECTS = {"1d": ("beam", "bandpass"),
+                  "2d": ("Tsky_mdl", ) }
+
+def _check_path(datafile):
+    # if the datafile is not an absolute path, assume it's in the data folder
+    if datafile[0] != "/":
+        datafile = path.join(DATA_PATH, datafile)
+    # make sure that the path exists
+    assert path.exists(datafile), \
+            "If datafile is not an absolute path, then it is assumed to " \
+            "exist in the hera_sim.data folder. The datafile passed could " \
+            "not be found; please ensure that the path to the file exists"
+    return datafile
+
+def _read_npy(npy):
+    return np.load(_check_path(npy))
 
 class Tsky:
```

Comment:
Since this is primarily for easily defining the Tsky from YAML, why not go the extra mile and make it a new YAML tag: https://stackoverflow.com/a/43060743/1467820? That way you should be able to define Tsky in any YAML by

```
tsky: !Tsky
    datafile: 'path/to/file.npz'
    pol: 'xx'
```

Reply:
It will automatically be loaded by `yaml.load` as an instance of `Tsky`.